Bio/Align/__init__.py

```diff
@@ -3770,8 +3770,15 @@ class PairwiseAligner(_pairwisealigner.PairwiseAligner):
     and gap scores are negative or zero.  By default, the match score is 1,
     and the mismatch and gap scores are zero.  Based on the values of the gap
     scores, a PairwiseAligner object automatically chooses the appropriate
-    alignment algorithm (the Needleman-Wunsch, Smith-Waterman, Gotoh, or
-    Waterman-Smith-Beyer global or local alignment algorithm).
+    alignment algorithm (the Needleman-Wunsch, Smith-Waterman, Gotoh,
+    Waterman-Smith-Beyer, or Fast Optimal Global Sequence Alignment Algorithm
+    global or local alignment algorithm).
```

Comment:
I guess this should be 

(the Needleman-Wunsch, Smith-Waterman, Gotoh, or Waterman-Smith-Beyer global or local alignment algorithm, or the Fast Optimal Global Sequence Alignment Algorithm).

Just for my curiosity, is there a version of FOGSAA for local alignments?

Reply:
I think one could theoretically adapt FOGSAA for local alignments, but I think it would almost always be slower than Smith-Waterman because FOGSAA would almost never be able to prune a branch. Also, the name is Fast Optimal *Global* Alignment Algorithm, so it really was never designed for local alignments.